generate_spectrum_index/result.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<interface id="speclib_add">

```

Comment:
It's a very old bad habit to just leave "speclib-add" as the "id" attribute value of this element. It seems that someone long ago copy/pasted their new workflow from the speclib-add workflow and never changed this. Since then this has propagated into countless new workflows.

This is admittedly not really used anywhere important. As far as I know its only use is in populating the workflow name in the LiveFlow flows.jsp web interface. But it is still potentially meaningful for human readers of this file and may be used in a more systematically meaningful way at some point in the future. Please make an effort to put the correct workflow identifier here just as you do in input.xml.